src/SFA.DAS.AssessorService.ApplyTypes/FinancialGrade.cs

```diff
@@ -9,10 +9,6 @@ public class FinancialGrade
         public string SelectedGrade { get; set; }
         public string InadequateMoreInformation { get; set; }
 
-        public FinancialDueDate OutstandingFinancialDueDate { get; set; }
-        public FinancialDueDate GoodFinancialDueDate { get; set; }
-        public FinancialDueDate SatisfactoryFinancialDueDate { get; set; }
```

Comment:
These are ViewModel classes and not part of the schema. Hence the major upheaval in the code changes above ☝️